src/metrics.rs

```diff
@@ -201,6 +201,12 @@ lazy_static! {
         exponential_buckets(256.0, 1.8, 22).unwrap()
     )
     .unwrap();
+    pub static ref ENGINE_WRITE_COMPRESSION_RATIO_HISTOGRAM: Histogram = register_histogram!(
+        "raft_engine_write_compression_ratio",
+        "Bucketed histogram of Raft Engine write compression ratio",
+        exponential_buckets(0.0005, 1.8, 16).unwrap()
```

Comment:
Does compression ratio fit such a wide range? In general, the ratio should be within 1~10, maybe  `exponential_buckets(0.5, 1.2, 16)` is enough? and it can provide higher precision.

Reply:
Here, the compression_ratio = `compressed_bytes_size / uncompressed_bytes_size`, not referring to Rocksdb, where the compression_ratio is calculated with `uncompressed_bytes_size / compressed_bytes_size`.

Reply:
Got it.